app/views/admin/application/_navigation.html.erb

```diff
@@ -21,4 +21,9 @@ as defined by the routes in the `admin/` namespace
     method: :delete,
     class: "navigation__link navigation__link--sign-out",
   ) %>
+  <%= link_to(
```

Comment:
in the tracker story the feature reads as 

```
WHEN I visit /admin and log in
THEN I should be prompted to complete two-factor authentication
```

my guess here is that someone can opt into 2FA but is not required to use it -- is there a way to make it required per the language used in the story?

Reply:
I enabled the mandatory option in the Devise initializer (enabled only for environments outside of "test")